pykafka/producer.py

```diff
@@ -17,17 +17,25 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-__all__ = ["Producer", "AsyncProducer"]
+__all__ = ["Producer", "SynchronousProducer"]
```

Comment:
What do you think about completely removing the `SyncProducer` class and having synchronous behavior switched on and off by a kwarg on `Producer`?